planner/planner_APIs.go

```diff
@@ -435,7 +439,10 @@ func (planner *MyPlanner) getPlanningApi(ctx *gin.Context) {
 	}
 	iowrappers.Logger.Debugf("[%s] The number of requested planning results is %s.", requestId, numResults)
 
-	planningReq := solution.GetStandardRequest(POI.Weekday(weekday), numResultsInt)
+	priceLevel := ctx.DefaultQuery("price", "2")
```

Comment:
So default level we always keep as `2` ?

Reply:
Yes, I kept the default with average price range value.